Home page restyling

JonWong37 opened the pull request:
1st issue branch upload for reviewing.

HowiJ commented:
1. Add a better summary to your pull request, include things like what you did, and include screenshots as proof of what you did (see [PR#4 Added Button Component](https://github.com/TeamDicey/dice-frontend/pull/4) and [PR#5 Added TextInput](https://github.com/TeamDicey/dice-frontend/pull/5) )
2. Better title, "1st issue test" insinuates that this is just a test or that this is testing something, however, I don't think that's what you meant to do in this PR

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  socket: Socket;
+}>;
+
+const Lobby = ({ socket }: Props) => {
```
rest of the codebase uses `function MyComponent(` as the syntax, lets' do that

JonWong37 replied:
Renamed line 12 to `function MyComponent(` and re-uploaded the file.

HowiJ reviewed (changes requested):
Review the comments, add screenshots to show what you've done, reasoning is cause I cannot see how it looks post changes so I can't tell if you've done what you need to do or not.

HowiJ commented on src/App.tsx:
```diff
@@ -4,6 +4,7 @@ import io from "socket.io-client";
 import LobbyForm from "LobbyForm";
```
this is unused right? remove if so

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  return (
+    <div>
+      <div className={css(styles.top)}>
+        <img className={css(styles.logo)} src={logo} alt="" />
```
alt="logo"

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  },
+  iRoll: {
+    fontWeight: 500,
+    fontSize: "16pt",
```
is pt the same as px? if it is, use 'px'

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  },
+  dicey: {
+    fontWeight: "bold",
+    fontSize: "40pt",
```
same as below

HowiJ commented on src/LobbyForm.tsx:
```diff
@@ -109,10 +109,10 @@ const hostStyles = StyleSheet.create({
     padding: "16px",
     borderRadius: "8px",
-    background: "#FFFFFF",
+    background: "#1D1E33",
```
does this still need a background? maybe we can remove it

HowiJ commented on src/LobbyForm.tsx:
```diff
@@ -109,10 +109,10 @@ const hostStyles = StyleSheet.create({
     margin: "auto",
     marginTop: "32px",
-    border: "1px solid #EEEEEE",
+    border: "1px solid #1D1E33",
```
same as above, do we need a border? rather no border than a border of the same color as the background

HowiJ reviewed (changes requested):
Yeah, let's fix the space on the bottom, figure out some way to make the entire background that color. Think about it this way, if you went to a website with a background and you only see the background color go about 60% of the way, would it be a janky website or a good website?

Generally, if there's something that's unnecessary, we should remove it rather than just changing it to the same values, IE if a thing shouldn't have a background, then don't have a background, same with border. 

Reasoning: 
As designs change, we should update the existing code as to not have too much old code that doesn't do anything to prevent issues popping up if designs change in the future

HowiJ commented:
noticed that the top bar is still there, probably want to remove that from the home page as well

HowiJ commented on src/App.tsx:
```diff
@@ -1,9 +1,9 @@
 import { css, StyleSheet } from "aphrodite";
 import io from "socket.io-client";
-import LobbyForm from "LobbyForm";
 import Profile from "Profile";
```
I think you can remove this right?

HowiJ commented on src/App.tsx:
```diff
@@ -73,6 +70,19 @@ const styles = StyleSheet.create({
   main: {
     display: "flex",
     flexDirection: "column",
+    backgroundColor: "#1D1E33",
+    height: "100vh",
```
I think rather than this, just do it on html or on body in the index.css

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  socket: Socket;
+}>;
+
+function Lobby({ socket }: Props): ReactElement | null {
```
doesn't need `| null`, you're never returning null (i know other files do this but just remove it here, we can remove other instances later)

HowiJ commented on src/Lobby.tsx:
```diff
@@ -0,0 +1,64 @@
+  words: {
+    textAlign: "center",
+  },
+  top: {
+    flexDirection: "column",
+    textAlign: "center",
+    justifyContent: "center",
```
does flexDirection or justifyContent do anything here? it's just a div wrapping an img right?

HowiJ commented on src/LobbyForm.tsx:
```diff
@@ -109,10 +109,8 @@ const hostStyles = StyleSheet.create({
     // minWidth: "230px",
```
whoops, i shouldn't have left this here, can you remove this for me? thanks

HowiJ reviewed (changes requested):
Pretty much good, just last few things

JonWong37 commented:
Resolved the Profile import not being used.
Moved background color to index.css. I left the vh in App.tsx because moving it to index.css adds a scrolling bar to our application.
Removed the null.
Removed the unused flex-direction and justify content.
Removed commented out code.

JonWong37 commented:
Submitted updated version of the App.tsx with cleaner code for the version text at the bottom
